### ISSUE TITLE
gestion grand nombre d'auteur

### ISSUE THREAD

**Issue by louis-ev** [l-atelier-des-chercheurs/dodoc#997]
Si plus de 10, affichage d'un picto pour raccourcir la liste : 

<img width="476" alt="image" src="https://github.com/user-attachments/assets/333a9ee7-8a39-4845-b7fc-77b6ded25bfa">

<img width="477" alt="image" src="https://github.com/user-attachments/assets/fda622e1-fc0f-45d9-ac91-50c76d3847c7">

**Comment by Spiderwan01** [comment]
Très bien, Est ce que l'image montre aussi que tu as pu remettre un outil qui aide à savoir qui est connecté et où en tant qu'admin ?
C'était un super outil de la V9 cela aussi !

**Comment by louis-ev** [comment]
Non… il faudrait remettre en place toute la logique, sur la base d'un petit descriptif des besoins (est-ce qu'on recense qqpart tous les utilisateurs présents et ou ils sont, est-ce qu'on l'indique aussi au niveau de la grille des projets, etc.)

Tu peux t'en charger @Spiderwan01 ? wishlist de ce qui te manque là-dessus dans une nouvelle issue, que ça eu existé dans la v9 ou pas ?
Je crois que @jubonhomme avait aussi des idées dans ce sens.

**Comment by Spiderwan01** [comment]
Je colle ça en todolist

Le jeu. 26 sept. 2024, 14:56, Louis Eveillard ***@***.***> a
écrit :

**Comment by jubonhomme** [comment]
Dans mes souvenirs de la V9 le suivi temps réel des auteurs utilisait beaucoup de ressources et faisait même planter l'instance quand on était une trentaine de participants.
N'ayant pas une utilisation en classe avec élèves, j'avoue que ça ne serait pas d'une grande utilité sur une plateforme dodoc comme celle de fablab à l'école. Mais peut-être qu'une utilisation en plus petit groupe comme une classe serait utile.

L'affichage du picto qui reprend le nombre des auteurs qui reste à afficher est bien vu. Ça reprend le fonctionnement des mots clés et autres métadonnées.

Par contre, une des demandes qui remonte souvent de mon côté c'est de pouvoir suivre les projets d'une classe (= un auteur) en particulier.
SI on est connecté, on a une vue directe sur ses 5 derniers projets et on peut filtrer avec le bouton "Mes projets". 
Mais on ne peut le voir que pour ses propres projets. 
Ça pourrait être utile de pouvoir filtrer les projets avec des noms d'auteurs. 
Dans l'affichage des filtres on pourrait avoir un filtre par auteur

**Comment by Spiderwan01** [comment]
Eh bien, ces outils v9 je les trouvais intéressants en effet en classe,
mais j’imagine  aussi pour animer une communauté de contributeurs dans
d’autres contextes, pour savoir : qui est là et qui travaille sur quoi
(notamment en  période hors temps de classe par ex pour une classe)  afin
d’affiner le suivi et l'accompagnement.

Savoir aussi en asynchrone,  qui est venu faire quoi ? Ça permet aussi de
voir quel projet  bénéficie d'un suivi, d'une fréquentation régulière et
quel projet est un peu oublié et peu revisité par ses auteurs. Cela  peut
aider l’admin à relancer,  a compagner, isoler une difficulté, être informé
rapidement d’une actualisation d’un contenu.

Et là je crois  si  j’administrais et animerais fablab à l’école je
trouverai cela bien pratique.

Le jeu. 26 sept. 2024, 22:26, jubonhomme ***@***.***> a
écrit :

**Comment by jubonhomme** [comment]
Oui je te rejoins sur ce point, je vois bien l'intérêt de suivre l'évolution du projet. 

Est-ce que l'écran d'accueil du projet ne pourrait pas servir de tableau de bord pour ce suivi.
Si on est simple visiteur on arrive directement sur les publications -> ça marche bien

Mais si on est dans le projet (référents ou contributeurs on arrive sur cet écran : 
![image](https://github.com/user-attachments/assets/c9a5774d-4ae8-4aba-8a1d-b15e4a89e602)


Pour le moment il y a juste l'info de la date de création et de dernière modification, ce n'est pas très informatif mais on avait déjà évoqué d'ajouter sur cet écran d'accueil les dernières modifications par exemple.

Je ne sais pas si le traçage temps réel des utilisateurs est un indicateur vraiment informatif. Ok on sait ou se trouve l'utilisateur mais ça ne dit pas grand chose sur ce qu'il fait ou produit. Je peux très bien rester connecté dix minutes sur une publication et ne rien ajouter, ou venir 10 fois sur un projet et ne rien faire dedans. 
Par contre si on affiche quelques indicateurs sur cet écran comme par exemple : 
12 nouveaux médias  -> si on clique dessus ça ouvre collecter
3 fabrications -> ça ouvre les dernières fabrications
2 publications modifiées -> ça ouvre les publications
...

Je trouve que ça permettrait de mieux suivre l'évolution d'un projet.
On ne trace pas les auteurs en temps réel mais on a quand même un suivi des modifications à l'accueil du projet.
